From c91e2f6d72823938052044eef858c3c675617ae4 Mon Sep 17 00:00:00 2001
From: Bruno Randolf <br1@einfach.org>
Date: Wed, 15 Mar 2017 21:22:48 +0000
Subject: [PATCH] Remove _GNU_SOURCE from ccan config

It creates problems with LEDEs FORTIFY and is not needed
---
 config.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/config.h b/config.h
index 5c398c4..7e1027a 100644
--- a/config.h
+++ b/config.h
@@ -1,8 +1,5 @@
 /* Generated by CCAN configurator */
 #ifndef CCAN_CONFIG_H
 #define CCAN_CONFIG_H
-#ifndef _GNU_SOURCE
-#define _GNU_SOURCE /* Always use GNU extensions. */
-#endif
 #define HAVE_TYPEOF 1
 #endif /* CCAN_CONFIG_H */
-- 
2.7.4

